miniupnpd/configure

```diff
@@ -760,6 +760,18 @@ case $OS_FAMILY in
 		;;
 esac
 
+# OpenSSL
+case $OS_FAMILY in
+	# OpenWrt disables https server for IGD v2
+	Linux)
+		if pkg_detect libssl; then
+			:
+		else
+			echo "Warning: libssl pkg-config not found!"
+		fi
```

Comment:
is there a reason to do that instead of ` if ! pkg_detect libssl ; then`  ?

Reply:
Purely aesthetical/readability reasons, such that the `if` branch is always the success path (throughout the configure script) and `else` branch is the error path.